No-GUI mode (--nogui) should not try to use DISPLAY

JabRef version 4.1 on RedHat 7 using JRE 1.8.0-162


Steps to reproduce:

1. use a headless (RH) Linux system that provides no DISPLAY
2. run only command line feature: 
3. java -jar JabRef-4.1.jar -n --help

<!-- If applicable, excerpt of the bibliography file, screenshot, and excerpt of log (available in the error console) -->

<details>
  <summary>JabRef does not run when invoked with the --nogui option and no DISPLAY is available.</summary>
  
  ```
Exception in thread "main" java.lang.reflect.InvocationTargetException
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:498)
        at sun.launcher.LauncherHelper$FXHelper.main(LauncherHelper.java:767)
Caused by: java.lang.UnsupportedOperationException: Unable to open DISPLAY
        at com.sun.glass.ui.gtk.GtkApplication.<init>(GtkApplication.java:68)
        at com.sun.glass.ui.gtk.GtkPlatformFactory.createApplication(GtkPlatformFactory.java:41)
        at com.sun.glass.ui.Application.run(Application.java:146)
        at com.sun.javafx.tk.quantum.QuantumToolkit.startup(QuantumToolkit.java:257)
        at com.sun.javafx.application.PlatformImpl.startup(PlatformImpl.java:211)
        at com.sun.javafx.application.LauncherImpl.startToolkit(LauncherImpl.java:675)
        at com.sun.javafx.application.LauncherImpl.launchApplicationWithArgs(LauncherImpl.java:337)
        at com.sun.javafx.application.LauncherImpl.launchApplication(LauncherImpl.java:328)
        ... 5 more
  ```
</details>

--- a/src/main/java/org/jabref/Globals.java
+++ b/src/main/java/org/jabref/Globals.java
@@ -1,5 +1,6 @@
 package org.jabref;
 
+import java.awt.GraphicsEnvironment;
 import java.awt.Toolkit;
 import java.util.Optional;
 import java.util.UUID;
@@ -77,7 +78,7 @@ public class Globals {
         Globals.fileUpdateMonitor = new DefaultFileUpdateMonitor();
         JabRefExecutorService.INSTANCE.executeInterruptableTask(Globals.fileUpdateMonitor, "FileUpdateMonitor");
 
-        if (Globals.prefs.shouldCollectTelemetry()) {
+        if (Globals.prefs.shouldCollectTelemetry() && !GraphicsEnvironment.isHeadless()) {
             startTelemetryClient();
         }
     }

